package-lock.json

```diff
@@ -0,0 +1,17669 @@
+{
```

Comment:
We're using yarn instead of npm! We shouldn't push this because we already have yarn.lock.

Reply:
when you approve this change can you discard of this file?

Reply:
> when you approve this change can you discard of this file?

and push back to master?

Reply:
yes 👍